streetwise/api/campaigns.py

```diff
@@ -29,6 +29,18 @@ def get(self):
 
 current_campaign = None
 
+def getCampaign(campaign_id):
+    campaign = None
+    if campaign_id is not None and campaign_id.isdigit():
+        campaign_id = int(campaign_id)
+        campaign = Campaign.query.filter(Campaign.id > campaign_id).first()
+    if campaign_id is None or campaign is None:
+        campaign = Campaign.query.first()
```

Comment:
Can we sort this?
`Campaign.query.order_by(Campaign.id).first()`

That way when the campaigns cycle there is no chance of sending the user the same campaign.

Reply:
Yes, I've added a sorting query.